mentat/parsers/replacement_parser.py

```diff
@@ -119,3 +120,47 @@ def _add_code_block(
             )
         )
         return ""
+
+    def file_edits_to_llm_message(
+        self, parsedLLMResponse: ParsedLLMResponse, git_root: Path | None = None
```

Comment:
I should have mentioned this in the last PR as well, but why are we passing in the git_root here? Why not always just use GIT_ROOT.get()? With mock_git_root the context variable should always exist in tests.

Reply:
I originally added it because I wanted to pass it in the translation script but then I ended up mocking it there too. I'll remove it.